pkg/container-collection/annotations.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package containercollection
+
+const (
+	// container types used by the runtimes
+	containerTypeSandbox   = "sandbox"
+	containerTypeContainer = "container"
+
+	// cri-o container annotations to get container information
```

Comment:
Could you add a comment saying where it comes from? I can see some of them at:

https://github.com/containers/podman/blob/main/pkg/annotations/annotations.go
https://github.com/containerd/containerd/blob/master/pkg/cri/annotations/annotations.go

But not all of them.